static/lang/re-en.json

```diff
@@ -1509,7 +1518,9 @@
                     "Note": "Each of your successful saving throws against a disease reduces its stage by 2, or by 1 for a virulent disease. Each critical success against an ongoing disease reduces its stage by 3, or by 2 for a virulent disease."
                 },
                 "GuidedByTheStars": {
-                    "Prompt": "Select saving throw or skill."
+                    "Prompt": "Select saving throw or skill.",
+                    "Save": "Save",
+                    "Skill": "Skill"
```

Comment:
Save exists in the ChoiceSet localization already, "Skill" strangely doesn't appear anywhere sensible, so let's add it to there as well.